tc/core/halide2isl.cc

```diff
@@ -93,7 +94,7 @@ namespace {
  * Convert Halide binary expression "op" into a list of isl affine functions by
  * converting its LHS and RHS into lists of affs and concatenating those lists.
  * This is intended to be used with Min/Max operations in upper/lower bound
- * computations, respectively.  Essentially, this allows for replacements
```

Comment:
I consistently use double spaces between sentences.  Makes it easy to read longer text fragments typeset in monospace fonts.  You may have noticed in my commit messages, or other comments, including this one.  I intend to keep doing so unless the code guidelines require the opposite.

Reply:
Me too.  In any case, this change does not belong to a commit that adds documentation.